controllers/module_reconciler.go

```diff
@@ -359,6 +375,7 @@ func (r *ModuleReconciler) SetupWithManager(mgr ctrl.Manager, kernelLabel string
 	return ctrl.NewControllerManagedBy(mgr).
 		For(&kmmv1beta1.Module{}).
 		Owns(&appsv1.DaemonSet{}).
+		Owns(&v1.ServiceAccount{}).
```

Comment:
Shouldn't we make the DaemonSet the owner of the serviceAccount herr?

Reply:
There are two ServiceAccounts per Module CR (one for the module loader and one for the device plugin) and both are owned ([module loader SA](https://github.com/kubernetes-sigs/kernel-module-management/pull/90/files#diff-98d75845082e0ae72cd6181df75be68f64cca9558e74b7122da4aadcfe0b0d0cR47) and [device plugin SA](https://github.com/kubernetes-sigs/kernel-module-management/pull/90/files#diff-98d75845082e0ae72cd6181df75be68f64cca9558e74b7122da4aadcfe0b0d0cR68)) by the Module CR and managed by the controller. 

Here, the controller-owned ServiceAccounts are setup to be watched, in order for the controller to respond to create / delete / update events on the former by reconciling the owner Module CR (and subsequently the ServiceAccounts themselves).

Does this make sense? Did I understand your question correctly?

Reply:
Thanks for the explanation. It does make a lot of sense :)